testonly/marshalling.go

```diff
@@ -27,12 +27,16 @@ import (
 // protobuf.ptypes.any.Any used throughout the Trillian API and in
 // storage.  Use if testing.T not available. Failure to marshal will
 // fail the test suite.
-func MustMarshalAnyNoT(in proto.Message) *any.Any {
+func MustMarshalAnyNoT(in proto.Message) []byte {
```

Comment:
Same here, could this simply be `proto.Marshal(in)`?

Reply:
done